fastestimator/op/tensorop/loss/cross_entropy.py

```diff
@@ -68,5 +90,11 @@ def forward(self, data: List[Tensor], state: Dict[str, Any]) -> Tensor:
                     form = "sparse"
             else:
                 form = "binary"
-        loss = self.cross_entropy_fn[form](y_pred, y_true, from_logits=self.from_logits, average_loss=self.average_loss)
+
+        class_dict = self.weight_dict["tf"] if tf.is_tensor(y_pred) else self.weight_dict["torch"]
```

Comment:
this can be a single thing populated during .build() rather than making a dictionary of both

Reply:
I will implement this.